src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -80,18 +80,18 @@ export const InitializeClient: FC<OwnProps> = ({
 
   useEffect(() => {
     dispatch(getBuckets())
-  }, []) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [dispatch])
 
   useEffect(() => {
     onSelectBucket(bucket.name)
   }, [bucket, onSelectBucket])
 
   useEffect(() => {
-    const fetchResources = async () => {
-      await dispatch(getAllResources())
+    const fetchResources = () => {
```

Comment:
Same as above - this led us to a nice bug we can squash in both instances where `getAllResources()` gets invoked.